### ISSUE TITLE
dphil application anonymization tool for dtc

### ISSUE THREAD

**Issue by martinjrobins** [OxfordRSE/website#65]
https://www.timeshighereducation.com/news/oxford-consider-socioeconomic-data-phd-applications

**Comment by iamleeg** [comment]
consider https://github.com/WealthWizardsEngineering/docs-anonymiser, I worked on a very similar problem at WW.

**Comment by martinjrobins** [comment]
yup, the tool is forked from that (https://github.com/OxfordRSE/docs-anonymiser)
